semcore/spin-container/src/style/spin-container.shadow.css

```diff
@@ -5,10 +5,10 @@ SSpinContainer {
   overflow: hidden;
 }
 
-SContent {
+/* SContent {
   z-index: 0;
   position: relative;
-}
+} */
```

Comment:
maybe we could remove this style?

Reply:
Oh, no, that's my fault – I commented it during debugging but should uncomment